docs/UserGuide.adoc

```diff
@@ -290,6 +290,8 @@ The student shares notes and questions with other students with Internet connect
 === Login using email and password `[coming in v2.0]`
 The student can register and login to the application using their email and password.
 
+=== Flash card grouping suggestions `[coming in v2.0]`
+NUStudy will intelligently suggest groupings of flash cards based on their content.
```

Comment:
Would it be better to elaborate a bit on the "intelligently"? Something like "intelligently suggest...by machine learning etc."

Reply:
Machine learning might not be needed.